### ISSUE TITLE
Multiclass input

### ISSUE THREAD

**Issue by marcojulioarg** [yingkaisha/keras-unet-collection#20]
Hello, I'm wondering how to do multiclass classification with Unet. I saw that a previous hot encode step is needed. My dataset consists in two classes + background. Thanks in advance!

**Comment by yingkaisha** [comment]
Sorry, the question is too general to be answered. 

Would this example help you? https://github.com/yingkaisha/keras-unet-collection/blob/main/examples/human-seg_atten-unet-backbone_coco.ipynb

**Comment by atgc1984** [comment]
My understanding is that if your pixel labels are something like 0/1/2 you may use _keras.losses.sparse_categorical_crossentropy_.
For _keras.losses.categorical_crossentropy_, the labels should be in one-hot format.

**Comment by marcojulioarg** [comment]
> 
> 
> My understanding is that if your pixel labels are something like 0/1/2 you may use _keras.losses.sparse_categorical_crossentropy_.
> For _keras.losses.categorical_crossentropy_, the labels should be in one-hot format.

Ok I understand, I was used to work with implementations that use (h,w,c) where c = number of classes. For this implementation a (h,w) mask with integer labeled pixels as classes. Thank you very much!

**Comment by yingkaisha** [comment]
Great to see the progress. Closing.
